src/main/scala/mesosphere/marathon/state/FetchUri.scala

```diff
@@ -21,13 +24,15 @@ case class FetchUri(
       .setExecutable(executable)
       .setExtract(extract)
       .setCache(cache)
-    outputFile.foreach{ name => builder.setOutputFile(name) }
+    outputFile.foreach { name => builder.setOutputFile(name) }
     builder.build()
   }
 }
 
 object FetchUri {
 
+  lazy val supportedFileTypes = Seq(".tgz", ".tar.gz", ".tbz2", ".txz", ".tar.xz", ".zip")
```

Comment:
Does this correspond to what mesos executor actually supports? http://mesos.apache.org/documentation/latest/fetcher/
We should update the list.